test/helpers/message-test-cases.js

```diff
@@ -0,0 +1,24 @@
+const { errorMessages, textMessages: { maxInfoLength, maxMessageLength } } = require('../../server/constants')
+
+module.exports = {
```

Comment:
These are shared between the message creation and message editing. Although it is a bit of pita to have them outside of the test file, sharing them between tests is really handy and means when one cases is added to one, it is added to both.